pkg/otlp/internal/serializerexporter/exporter.go

```diff
@@ -40,16 +56,52 @@ type exporter struct {
 	s  serializer.MetricSerializer
 }
 
-func newExporter(logger *zap.Logger, s serializer.MetricSerializer) (*exporter, error) {
-	// TODO (AP-1267): Expose these settings in datadog.yaml.
-	tr, err := translator.New(logger,
+func translatorFromConfig(logger *zap.Logger, cfg *exporterConfig) (*translator.Translator, error) {
+	histogramMode := translator.HistogramMode(cfg.Metrics.HistConfig.Mode)
+	switch histogramMode {
+	case translator.HistogramModeCounters, translator.HistogramModeNoBuckets, translator.HistogramModeDistributions:
+		// Do nothing
+	default:
+		return nil, fmt.Errorf("invalid `mode` %s", cfg.Metrics.HistConfig.Mode)
```

Comment:
```suggestion
		return nil, fmt.Errorf("invalid `mode` %q", cfg.Metrics.HistConfig.Mode)
```
(so that it is inside quotes)